arbor/cable_cell_param.cpp

```diff
@@ -152,4 +152,37 @@ void decor::set_default(defaultable what) {
             what);
 }
 
+std::ostream& operator<<(std::ostream& os, const paintable& item) {
```

Comment:
FYI, we have a helper in `util` to make this code less verbose: 
```
auto paintable_visitor = arb::util::overload(
     [&](const init_membrane_potential& p) { os << "init-membrane-potential" },
     [&](const axial_resistivity & p) { os << "axial-resistivity"; },
     [&](const temperature_k & p){ os << "temperature-kelvin"; }, 
     ...
);
std::visit(paintable_visitor, item);
```
However, we don't use it in other spots in this file, so it might be better to stick to the same style.

Reply:
I find the `if constexpr (...)` style slightly better, if it's the same to you.